fury/ui/core.py

```diff
@@ -325,6 +327,41 @@ def mouse_move_callback(i_ren, obj, self):
     @staticmethod
     def key_press_callback(i_ren, obj, self):
         self.on_key_press(i_ren, obj, self)
+    
+    @staticmethod
+    def window_event_propagate(window, evt, obj, i_ren):
+        """Method to propagate a windows event to a specific actor.
+
+        Parameters
+        ----------
+        window: :class: `vtkWin32OpenGLRenderWindow`
+            Current window
+        evt: str
+            Window invoked event
+        obj: :class: `UI`
+            UI element where event isto be propagated
+        i_ren: :class: `CustomInteractorStyle`
+            Custom Interactor associated with the window
+        """
+        if not hasattr(obj, 'actor'):
+            raise AttributeError(
+                f'{obj} doesnt have an actor associated with it')
```

Comment:
just `getattr(obj, 'actor')` for this entire block will work.

Reply:
Just provides a more insightful error message.